src/app/core/enums/active-route-nav.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum ActiveRouteNavigationEnum {
+  CharactersRoute,
```

Comment:
it is not necessary to use 'Route' in each enum value

Reply:
I've changed them